src/services/TerminalCommandExecuter.ts

```diff
@@ -74,16 +74,40 @@ export class TerminalCommandExecuter {
         iconPath: icon ? new ThemeIcon(icon) : undefined
       });
 
+      // Check the user's settings to see if they want to use nvm or nvs
+      // Get the user's preferred node version manager -- nvm or nvs or none, if they don't want to use either
+      const nodeVersionManager: string = TerminalCommandExecuter.getExtensionSettings('nodeVersionManager', 'nvm');
+
       // Check if nvm is used
       const nvmFiles = await workspace.findFiles('.nvmrc', '**/node_modules/**');
-      if (nvmFiles.length > 0) {
-        terminal.sendText('nvm use');
+
+      // If there are .nvmrc files and the user wants to use nvm, then use their preferred node version manager
+      if (nvmFiles.length > 0 && nodeVersionManager !== 'none') {
+
+        // Launch the command to use the node version specified in the .nvmrc file
+        if (nodeVersionManager === 'nvs') {
+          // Use nvs
+          terminal.sendText('nvs use');
+        } else {
+          // Default to nvm
+          terminal.sendText('nvm use');
+        }
```

Comment:
in `constants` folder lets create a new enum `NodeVersionManagers` like

```
export enum NodeVersionManagers {
  nvm = 'nvm',
  nvs = 'nvs',
  none = 'none'
}
```

and use it here, also let's use it in line 79 like 
`const nodeVersionManager: string = TerminalCommandExecuter.getExtensionSettings('nodeVersionManager', NodeVersionManagers.none);`

also lets remove the not needed comments
```suggestion
      // If there are .nvmrc files and the user wants to use nvm, then use their preferred node version manager
      if (nvmFiles.length > 0 && nodeVersionManager !== NodeVersionManagers.none) {
        if (nodeVersionManager === NodeVersionManagers.nvs) {
          terminal.sendText('nvs use');
        } else {
          terminal.sendText('nvm use');
        }
      }
```